--- a/ois/trunk/includes/OISPrereqs.h
+++ b/ois/trunk/includes/OISPrereqs.h
@@ -89,6 +89,7 @@
 #   define _OISExport __attribute__((visibility("default")))
 #else //Probably Linux
 #	define OIS_LINUX_PLATFORM
+#	include <unistd.h>
 #endif
 
 //Is Processor 32 or 64 bits...
@@ -115,10 +116,10 @@
 	class Keyboard;
 	class Mouse;
 	class JoyStick;
-    class MultiTouch;
+	class MultiTouch;
 	class KeyListener;
 	class MouseListener;
-    class MultiTouchListener;
+	class MultiTouchListener;
 	class JoyStickListener;
 	class Interface;
 	class ForceFeedback;
@@ -135,7 +136,7 @@
 	typedef std::map<Object*, FactoryCreator*> FactoryCreatedObject;
 
 	//! Each Input class has a General Type variable, a form of RTTI
-    enum Type
+	enum Type
 	{
 		OISUnknown       = 0,
 		OISKeyboard      = 1,
@@ -151,7 +152,7 @@
 	//--------     Shared common components    ------------------------//
 
 	//! Base type for all device components (button, axis, etc)
-    enum ComponentType
+	enum ComponentType
 	{
 		OIS_Unknown = 0,
 		OIS_Button  = 1, //ie. Key, mouse button, joy button, etc

