diff -Naur qcad-3.4.2.0.orig/support/examples/transactionlistener/transactionlistener.pro qcad-3.4.2.0/support/examples/transactionlistener/transactionlistener.pro
--- qcad-3.4.2.0.orig/support/examples/transactionlistener/transactionlistener.pro	2013-10-25 13:28:49.000000000 +0000
+++ qcad-3.4.2.0/support/examples/transactionlistener/transactionlistener.pro	2013-10-28 02:41:57.269128756 +0000
@@ -5,5 +5,5 @@
 TEMPLATE    = lib
 HEADERS     = TransactionListenerPlugin.h
 SOURCES     = TransactionListenerPlugin.cpp
-DESTDIR     = ../../../../qcad/plugins
+DESTDIR     = ../../../plugins
 LIBS += -lqcadcore -lqcadgui
